tests/TestCase.php

```diff
@@ -250,7 +250,7 @@ protected static function getInvalidDocumentCodecValues(): array
      */
     protected static function getInvalidHintValues(): array
     {
-        return [123, 3.14, true];
+        return [123, 3.14, true, PackedArray::fromPHP(['hint'])];
```

Comment:
```suggestion
        return [123, 3.14, true, PackedArray::fromPHP([])];
```

Assuming this passes all tests just the same, I don't think a value matters.

Reply:
Indeed, it can be confusing to put a value.